docs/fast_data/single_view_trigger_generator.md

```diff
@@ -8,29 +8,59 @@ sidebar_label: Single View Trigger Generator
 This Plugin is a BETA Plugin and, as such, is currently under active development. Pay attention using it.
 :::
 
-The Single View Trigger Generator is a new service of the Fast Data architecture.
-Its main job is to execute the strategies which are currently handled by the [Real Time Updater](/fast_data/realtime_updater.md).
+The Single View Trigger Generator is a new component of the Fast Data architecture. Its introductions allow to split the import phase
+from the [strategies](/fast_data/the_basics.md#strategies) execution, which currently are both handled by the [Real Time Updater](/fast_data/realtime_updater.md).
 
-Here's how the basic Fast Data lifecycle looks like with the Single View Trigger Generator:
+Below is provided a diagram showing how the Single View Trigger Generator service integrates within Fast Data flow:
 
-![Fast data lifecycle with Single View Trigger Generator](img/svtg-diagram.png)
+![Fast data lifecycle with Single View Trigger Generator](img/svtg-fd-arch.svg)
 
-Specifically, the service takes a [Projection Update message](/fast_data/inputs_and_outputs.md#projection-update) sent by the Real Time Updater and executes the strategy generating an identifier. From that identifier the service sends a [Projection Change message](/fast_data/inputs_and_outputs.md#single-view-trigger) which will be read by the [Single View Creator](/fast_data/single_view_creator.md) triggering the aggregation of the Single View related to the identifier.
+In this particular Fast Data configuration, Real-Time Updater is set to not execute projection changes strategies, but to rather emit a [Projection Update event](/fast_data/inputs_and_outputs.md#projection-update) (also known as `pr-update`) for each modified projection.  
+The Single View Trigger Generator reacts to these events executing the corresponding projection change strategy. The strategy produces one or more identifiers of the Single View that needs to be updated.
+This information is then passed over to the downstream component through [Single View Trigger events](/fast_data/inputs_and_outputs.md#single-view-trigger) (also known as `sv-trigger`), one for each produced Single View identifier.  
+Eventually, the [Single View Creator](/fast_data/single_view_creator.md) consumes these events triggering the corresponding logic for aggregating the interested Single View.
 
-### Main differences with the current architecture
+### Advantages of adopting the Single View Trigger Generator
```

Comment:
Let's add the advantages also of a full event-driven architecture

Reply:
Done